File: PhamaMicroCRM/src/PhamaMicroCrm.Model/Entities/Company.cs
using System;
using System.Collections.Generic;

namespace PhamaMicroCrm.Model.Entities
{
    public class Company : Entity
    {
        public string Name { get; set; }
        public string Field { get; set; }
        public bool Active { get; set; }
        public string Phone { get; set; }
        public DateTime CreatedAt { get; set; }

        public IEnumerable<CompanyUnit> CompanyUnits { get; set; }
    }
}

File: PhamaMicroCRM/src/PhamaMicroCrm.Web/wwwroot/js/Entities/Company.js
class Company {
    constructor(name, field, active, phone) {
        this.name = name;
        this.field = field;
        this.active = active;
        this.phone = phone;
    }

    //methods
    greet() {
        console.log(`Hi, I'm a company called ${this.name}`);
    }   
}